material/material-theme-text-field.html

```diff
@@ -0,0 +1,121 @@
+<dom-module id="material-theme-text-field" theme-for="vaadin-text-field">
+  <template>
+    <style>
+      :host {
+        position: relative;
+        width: 160px;
+        padding-top: 16px;
+        margin-bottom: 8px;
+        color: rgba(0, 0, 0, 0.87);
+        -webkit-font-smoothing: antialiased;
+        -moz-osx-font-smoothing: grayscale;
+
+        /* Fix Chrome’s dancing labels */
+        contain: content;
+
+        /*
+        -webkit-transition needed for Safari right now, but it’s fixed in
+        Safari Technology Preview, so we can expect it to fix itself in the
+        near future (Safari 10.2 most probably).
+        */
+        -webkit-transition: background-size 0.25s;
+        transition: background-size 0.25s;
+
+        background-size: 0% 2px, 100% 2px;
+        background-image: linear-gradient(#3F51B5, #3F51B5), linear-gradient(rgba(0, 0, 0, 0.12) 1px, transparent 1px);
```

Comment:
Underline color is too low contrast according to the specs. Should be 0.42 instead.